public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.tsx

```diff
@@ -116,7 +116,9 @@ export function MetricStatEditor({
                 if (
                   !statistic ||
                   (!standardStatistics.includes(statistic) &&
-                    !/^p\d{2}(?:\.\d{1,2})?$/.test(statistic) &&
+                    !/^(p\d{2}(?:\.\d{1,2})?|tc|ts|wm|tm|pr)(\(\d+(?:\.\d+)?%?:\d+(?:\.\d+)?%?\)|:\d+(?:\.\d+)?)?$/.test(
+                      statistic
+                    ) &&
```

Comment:
I did a quick test and found that this regex doesn't cover the 6 new statistics.

4 of the stats, 1. Trimmed Mean (TM), 2. Trimmed Count (TC), 3. Trimmed Sum (TS), and 4. Windsorized Mean (WM), follow the syntax described here https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Statistics-definitions.html#Statistics-syntax

For the remaining 2:
- Percentile rank: follows the form `PR(X:Y)`, `PR(X:)`, or `PR(:Y)`
- Interquartile mean: is just `IQM`. This can be added to https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/cloudwatch/standardStatistics.ts since `IQM` doesn't have any parameters.

Please note that the capitalization of the statistic matters, e.g. the form `tm(X:Y)` is invalid. The linked documentation mentions the following:

> They can only be abbreviated using lowercase letters when you specifying only one number, such as **tm99**.

Reply:
Updated now.